codec_specs.md

```diff
@@ -561,16 +561,17 @@ Codec ID: A_AC3
 
 Codec Name: Dolby Digital AC-3
 
-Description: BSID <= 8 !! The private data is void ??? Corresponding ACM wFormatTag : 0x2000 ; channel number have
-to be read from the corresponding audio element
+Description: This corresponds to a bitstream with the value of the "Bit Stream Identification" `bsid` in the "Bit Stream Information" structure set to 8 as defined in Section 5.4.2.1 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366].
+Corresponding ACM wFormatTag : 0x2000 ; channel number have to be read from the corresponding audio element
 
 ### A_AC3/BSID9
 
 Codec ID: A_AC3/BSID9
 
 Codec Name: Dolby Digital AC-3
 
-Description: The ac3 frame header has, similar to the mpeg-audio header a version field. Normal ac3 is defined as bitstream id 8 (5 Bits, numbers are 0-15).
+Description: This corresponds to a bitstream with the value of the "Bit Stream Identification" `bsid` in the "Bit Stream Information" structure set to 9.
+The `bsid` is defined in Section 5.4.2.1 of [@!ATSC-A52] or Section 4.4.2.1 of [@!ETSI-TS102-366].
```

Comment:
I generally think that we shouldn't list `A_AC3/BSID<n>` as separate `CodecID`s & collapse all `A_AC3(/BSID<n>)` into a single section. Together with Jérôme's last proposed wording in the section above this should be fine, and it would make reading the section quicker. It would also make it easier to understand what the difference between `A_AC3` and `A_AC3/BSID<n>` as: none except for the `CodecID`, making the `/BSID<n>` suffix completely information & historical.